server/neptune/gateway/deploy/requirement/aggregate.go

```diff
@@ -63,6 +63,20 @@ func NewAggregate(cfg valid.GlobalCfg, teamFetcher *github.TeamMemberFetcher, re
 	)
 }
 
+func NewPRAggregate(globalCfg valid.GlobalCfg) *Aggregate {
```

Comment:
Are we going to need more requirements here?

Reply:
Reason I asked is because it's more common to have requirements that need to be fullfilled before allowing a deploy, but a PR event shouldn't be mutating state.  Also this package lives under deploy as well and `Aggregate` looks for force apply commands which doesn't even apply here.  Would rather not shoehorn if we don't need to.

Reply:
These 3 requirements come from me translating the original autoplan validator [logic](https://github.com/lyft/atlantis/blob/release-v0.17.3-lyft.1/server/neptune/gateway/event/autoplan_builder.go#L55) into the new world. I didn't notice the package lived under deploy which we can move out or I can just duplicate some of the logic if that's cleaner.

Reply:
I'd rather just keep this logic in the legacy handler for now, you don't have to go this far with it, especially if it's being removed.

Reply:
or just don't reuse `Aggregate` because it checks for a force apply here which doesn't really apply.